### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by imanner** [getlantern/lantern-forum-cn#1099] [General]
从开通了当天就问题不断，于当天第二天我都提交了带有报告的问题，现在也没有解决
vpn地址老自动切换，我传东西老断, 对于我的业务，我完全没办法正常使用
由于实在无法满足需求，特申请退款
大家都不容易，如果给团队带来困扰，深感抱歉

**这是我提交的问题，里面有错误报告**
![2024-12-22_224923](https://github.com/user-attachments/assets/18c38c43-29c0-4cfe-9204-65ae02f61f25)

**付款记录**
![2024-12-22 223650](https://github.com/user-attachments/assets/a63cd2b0-9520-4d70-aba2-24cc35e6dd14)

我的账号 **wowiwo@yeah.net**

最后，还是感谢蓝灯团队，此次无缘了，希望团队能改良产品，有机会，以后再见

**Comment by imanner** [comment]
管理员如果收到信息请周知一下，谢谢。

**Comment by imanner** [comment]
退款事项也要等美国团队放假回来吗？
如果是这样，请管理员注意：
- 我是在试用第7天的时候申请的退款，如果是因你们没法办公引起的延时，应该从我申请之日算起，具体原因如下条，
- 从申请退款时间【也就是2024-12-22 23:00 】起，我不再使用蓝灯的付费版本，这个你们应该能查到，
- 在上一条的前提下，除非你们要求我提交付费版本的什么日志数据，否则遵循上一条不再使用
- 如果你们停止了我的专业账户服务，应视为你们同意停止服务并退款
- 我同意支付这7天使用的费用，但清源路退回款项，由于支付宝分期会产生相关费用，我可以另行支付相关费用
- 为避免支付宝分期带来的处理问题，请及时处理

以上，请管理员看到后，积极处理，谢谢

**Comment by MaxMKMax** [comment]
@imanner 您好. 如果您已退回免费版的话应该是已经帮您弄着退款了. 请您耐心等待谢谢您.

现在美国团队都在放假不在线呢. 要等一周后才回来了. 到时候我再帮您追问好嘛? 非常抱歉让您久等了. 谢谢您.

**Reply by imanner** [reply]
我试了下，还没退回到免费版。
到时候麻烦您帮我追问一下了，先谢过。

**Reply by imanner** [reply]
等到方便的时候，请把进展告诉我，谢谢

**Comment by MaxMKMax** [comment]
@imanner 您好. 已经帮您提交上去处理了. 请您留意邮箱哦. 谢谢您的耐心等待.

**Reply by imanner** [reply]
谢谢

**Reply by imanner** [reply]
请问多久 能完成退款，马上一周了，到什么进度了？

**Reply by imanner** [reply]
??????????????????????????????????????????
